dojo/endpoint/views.py

```diff
@@ -397,14 +397,21 @@ def endpoint_bulk_update_all(request, pid=None):
                 if skipped_endpoint_count > 0:
                     add_error_message_to_response('Skipped mitigation of {} endpoints because you are not authorized.'.format(skipped_endpoint_count))
 
+                eps_count = 0
+
                 for endpoint in endpoints:
-                    endpoint.mitigated = not endpoint.mitigated
-                    endpoint.save()
+                    for eps in endpoint.endpoint_status:
```

Comment:
might be good to prefetch the endpont_status relation to make it performant on large number of endpoints?

Reply:
Hmmm... and maybe use of [`.update()`](https://docs.djangoproject.com/en/4.0/ref/models/querysets/#django.db.models.query.QuerySet.update) would be the best idea

Reply:
@kiblik are you planning on changing this so we can merge it?

Reply:
I will prepare it today/tomorrow.